src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs

```diff
@@ -281,34 +281,31 @@ public BehaviorOnContinueAsNew EventBehaviourForContinueAsNew
 
         #region IEntityOrchestrationService
 
-        EntityBackendProperties IEntityOrchestrationService.GetEntityBackendProperties()
+        EntityBackendProperties IEntityOrchestrationService.EntityBackendProperties
            => new EntityBackendProperties()
            {
                EntityMessageReorderWindow = TimeSpan.FromMinutes(this.settings.EntityMessageReorderWindowInMinutes),
                MaxEntityOperationBatchSize = this.settings.MaxEntityOperationBatchSize,
                MaxConcurrentTaskEntityWorkItems = this.settings.MaxConcurrentTaskEntityWorkItems,
                SupportsImplicitEntityDeletion = false, // not supported by this backend
                MaximumSignalDelayTime = TimeSpan.FromDays(6),
+               UseSeparateQueueForEntityWorkItems = this.settings.UseSeparateQueueForEntityWorkItems,
            };
 
-        bool IEntityOrchestrationService.ProcessEntitiesSeparately()
-        {
-            if (this.settings.UseSeparateQueueForEntityWorkItems)
-            {
-                this.orchestrationSessionManager.ProcessEntitiesSeparately = true;
-                return true;
-            }
-            else
-            {
-                return false;
-            }
-        }
+        EntityBackendQueries IEntityOrchestrationService.EntityBackendQueries
+           => new EntityTrackingStoreQueries(
+                this.messageManager,
+                (this.trackingStore as AzureTableTrackingStore)
+                    ?? throw new NotSupportedException("entity queries not supported for custom tracking stores"),
```

Comment:
nit:
```suggestion
                    ?? throw new NotSupportedException("entity queries are not supported for custom tracking stores"),
```

Also, why separate tracking stores not supported?

Reply:
Accessing the internal methods of AzureTableTrackingStore is a bit more efficient. However, now that I look at it again, it seems not very hard to add a little bit of code to make it work with tracking stores also.